setup.py

```diff
@@ -23,10 +23,15 @@
         'catkit.pawprint',
         'catkit.gen',
         'catkit.gen.api',
-        'catkit.flow'
+        'catkit.flow',
+        'catkit.hub',
+        'catkit.hub.ase_tools'
     ],
     package_dir={'catkit': 'catkit'},
-
+    entry_points='''
```

Comment:
If your console scripts are not python scripts they will need to be added to the MANAFEST.in file to be included.

Reply:
I didn't see any on-py files so this is a moot point.

Reply:
So I have to add my catkit/hub/readme.org?

Reply:
Yes, if you want them to be included.